app/assets/stylesheets/footer.scss

```diff
@@ -0,0 +1,19 @@
+@import "variables";
+
+footer {
+  box-sizing: border-box;
```

Comment:
Properties should be ordered background, box-sizing, clear, color, font-size, height, line-height, margin-top, padding, position